### DISCUSSION TITLE
guide on alpine uefi zfs needs minor changes

### DISCUSSION THREAD

**Discussion by Denyreal** [zbm-dev/zfsbootmenu#654] [General]
Hey fellows hope you're doing well.

Just tried to apply ZFSbootmenu alpine uefi with zfs encrypted. It works like a charm.

Two things apparently missing from current documentation . 

- when chrooted i needed to specify 'nameserver 9.9.9.9' in /etc/resolv.conf to be able to ... (it's always dns)
- when chrooted i needed to redefine the BOOT_DEVICE variables that was only defined in live iso env

hope it helps

i'm trying to automate this with an ansible role. wish me good luck . 

regards.

**Comment by zdykstra** [comment]
Are you seeing these issues when doing the installation by hand, or as part of your Ansible role development?

**Reply by Denyreal** [reply]
the manual installation . 

The most tricky stuff for ansible role is the chroot thing i guess.

**Comment by Denyreal** [comment]
By the way would be great if there's some kind of dropbear remote decrypt process to implement.

**Reply by zdykstra** [reply]
https://docs.zfsbootmenu.org/en/v2.3.x/general/remote-access.html

It'll require building in a container on Alpine.

**Reply by Denyreal** [reply]
amazing thanks !!!
